crates/torii/grpc/proto/world.proto

```diff
@@ -4,7 +4,17 @@ package world;
 // The World service provides information about the world.
 service World {
     // Retrieves metadata about the world.
-    rpc Meta (MetaRequest) returns (MetaReply);
+    rpc Meta (MetaRequest) returns (MetaResponse);
+    // Retrieves the component values for an entity.
+    rpc GetEntity (GetEntityRequest) returns (GetEntityResponse);
+    // Retrieves all entities of a component.
+    rpc GetEntities (GetEntitiesRequest) returns (GetEntitiesResponse);
+
+    /* 
+     * Subscribes to entity updates.
+     * Bidirectional streaming as we want to allow user to change the list of entities to subscribe to without closing the connection.
+     */
+    rpc SubscribeEntities (stream SubscribeEntitiesRequest) returns (stream SubscribeEntitiesResponse);
```

Comment:
probably better to have this in a separate service

Reply:
Hmm I think it is fine to include here